CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Smithy Changelog
 
+## 1.52.0 (2024-10-15)
+
+### Features
```

Comment:
All the PR links aren't actual markdown links, they're only rendered that way on GitHub because it has a custom check for things that match issue/PR IDs.

Reply:
Forgot to do that, fixed.